### ISSUE TITLE
[Bug] Broken manual search

### ISSUE THREAD

**Issue by hotio** [morpheus65535/bazarr#163]
**Describe the bug**
If for a movie no subtitles are missing, a manual search does not show any results, even when plenty of results can be found on opensubtitles.org.

**To Reproduce**
1. Go to 'movie page for movie that has no missing subtitles'
2. Click on 'manual search'
3. No results are found

**Expected behavior**
Show all the results and let the user download a subtitle that overwrites the existing subtitle

**Additional context**
At the moment I can only confirm this for the movies section, currently am testing without Sonarr support enabled to eliminate running accidentally into an API ban for one of the providers.

**Comment by hotio** [comment]
I can confirms that this only applies to movies

**Comment by morpheus65535** [comment]
I can't reproduce it. You must be missing something in your reproduce steps. My manual search work perfectly for a movie that don't miss any subs.

**Comment by hotio** [comment]
![manual](https://user-images.githubusercontent.com/26902309/46245980-642fe900-c3f7-11e8-8b04-2f184f2e3c56.png)

**Comment by hotio** [comment]
Indeed, it just doesn't find anything for certain movies

**Comment by morpheus65535** [comment]
LOL ok that one is going to be fun to debug... I don't even know where to start...

**Comment by hotio** [comment]
Plenty of results are found https://www.opensubtitles.org/nl/search/sublanguageid-dut,eng/idmovie-33485
lol

**Comment by morpheus65535** [comment]
In debug logging, you see something?

**Comment by hotio** [comment]
I see a lot of stuff flash by, somewhat the same as when it does find stuff

**Comment by morpheus65535** [comment]
Can you clear the log and send try it again? I would like to have it.

**Comment by hotio** [comment]
will do, an auto search for another movie found the subtitle, manual search does not show anything

**Comment by hotio** [comment]
This is for the movie 47 Meters Down, Bazarr is restarted, refreshed the movie page, pushed manual search, until popup says 'No subtitles found'.
https://pastebin.com/v3pgrZUy

A click on the blue button for a missing sub did work and downloaded a subtitle.
Enabled providers are Addi7ed, thesubdb and opensubtitles

**Comment by morpheus65535** [comment]
Ok thank I'll look into it tonight. Family time first.

**Comment by morpheus65535** [comment]
Your movies is detected as a series by GuessIt. No way it could return a matching subs. In automatic search, I don't require matching title to match. In manual search, I do. If you check the downloaded subs with automatic search, it probably just don't match your movie.

**Comment by hotio** [comment]
```
For: 47 Meters Down (2017) - [BLURAY-1080P][DTS 5.1][X264]-BLOW.
GuessIt found: {
    "title": "47 Meters Down", 
    "year": 2017, 
    "format": "BluRay", 
    "screen_size": "1080p", 
    "audio_codec": "DTS", 
    "audio_channels": "5.1", 
    "video_codec": "h264", 
    "release_group": "BLOW", 
    "type": "movie"
}

```

**Comment by hotio** [comment]
Why would bazarr even allow guessit to try and guess a movie pulled from radarr guessing as a series? guessit supports `--type`.

**Comment by morpheus65535** [comment]
First, your not up to date with GuessIt. The version used with Bazarr is 3.0.0. Second, at the time I've written the automatic search, I've used `download_best_subtitle` from subliminal that do not give me the opportunity to exclude result between the scoring process and the download event. I want to go back to this part of the code in the upcoming future to implement Asyncprovider and put minimal criteria (title for movies; series, season and episode for series). All of this require some coding and testing and, btw, Bazarr is my hobby, I've got full time work, wife and kids and house to attend. I'm not some unemployed dude living in his parent's basement that have nothing else to do...

**Comment by hotio** [comment]
Kind of a strange response....Are you suggesting I should upgrade to get the bug? Are you telling me that I'm an "unemployed dude living in his parent's basement that have nothing else to do..."?

**Comment by morpheus65535** [comment]
No, not at all, sorry for the misunderstanding. I just do my best here to give the best but you seems to be searching for some little bug and, to be honest, your passive aggressive tone in your comment is driving me mad. BTW, I've got comment on Discord about it of other users so I don't think it's my imagination that is playing me tricks here.

So, please, can you explain in one simple post your issue and I'll see if I can reproduce it and fix it?

**Comment by hotio** [comment]
Seriously, I'm not trying to be passive/agressive, I'm just trying to help you out make bazarr the best subtitle search program there is. Therefor I installed bazarr and reported what I 'thought' were bugs after 30 minutes of testing and some feature requests that might have been usefull.

I do feel as you don't take anything I say serious and just shrug off everything as an upstream bug, so if I'm talking bullocks please feel free to close all my issues, no point in wasting my time if it falls on deaf ears.

**Comment by morpheus65535** [comment]
Ok, let's start from scratch then. What is the issue you're facing? I'm kind of lost in all those reply.

BTW, thank you for reporting bugs. I appreciate you taking of your time to do so.

**Comment by morpheus65535** [comment]
Should be fixed in 0.6.6. Thanks.
